doc/source/api/sequence_annotations.rst

```diff
@@ -0,0 +1,46 @@
+.. _sequence_annotations:
+
+************************
+Sequence Annotations API
+************************
+For the Sequence Annotation schema definitions, see `Sequence Annotation schema <../schemas/sequenceAnnotations.html>`_
+
+
+------------------------
+Feature Based Hierarchy
+------------------------
+The central object of the GA4GH Sequence Annotation API is a Feature.  The Feature describes an interval of interest on some reference(s).  It has a span from a start position to a stop position as well as descriptive data.  A Feature has one parent Feature, and can have an ordered array of child Features, which enables the construction of more complex representations in a hierarchical way.
```

Comment:
Should this be 'A Feature can have one parent Feature' rather than 'A Feature has one parent Feature'?